dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -3809,7 +3809,7 @@ paths:
         format: "int64"
       - name: "versionName"
         in: "query"
-        description: "Version ID"
+        description: "Version Name"
```

Comment:
Just curious, did you manually update this? I didn't think we were building Swagger any more.

Reply:
Yeah, I changed this manually but I suppose it's redundant if swagger isn't being built.